### PR TITLE
S5 deployment

### PR DESCRIPTION
**Pull request by PapaPwn**
Just need to wait for @Aruzal and @DarkPurple141 PRs to be merged, then I'll merge dev.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,16 +7,38 @@ notifications:
...
+        - cd ../
+      after_script:
+        - cd backend
+        - npm run coverage
+        - cd ../
+
+    - stage: deploy-dev
+      script:
+        - ./deploy.sh staging
+      if: branch = dev
```
Comment (DarkPurple141) [inline .travis.yml]: neato

File: frontend/src/utils/api/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import APIError from './errors'
-// TODO eventually second URL should be set to deploy url
-const API_URL = process && process.env
-  ? 'http://localhost:3000/api' : 'http://127.0.0.1:3000/api'
+// TODO - THIS DOESN'T SEEM TO BE WORKING...
+export const API_URL = process && process.env && process.env.NODE_ENV === 'development'
```
Comment (DarkPurple141) [inline frontend/src/utils/api/index.js]: is there a reason this is now export?

Comment (PapaPwn) [reply frontend/src/utils/api/index.js]: Nope

Comment (PapaPwn) [reply frontend/src/utils/api/index.js]: Originally it was because of the AppSearch component having the hardcoded URL

**Pull request merged**